Tests/MapboxCoreNavigationTests/MapboxCoreNavigationTests.swift

```diff
@@ -36,7 +36,7 @@ class MapboxCoreNavigationTests: TestCase {
     }
     
     func testNavigationNotificationsInfoDict() {
-        navigation = MapboxNavigationService(route: route,
+        navigation = MapboxNavigationService(routeResponse: response,
```

Comment:
It'd be great to create a follow-up ticket to update documentation everywhere. In scope of this PR files like `README.md` and `cover.md` can be updated as they show how to use `NavigationViewController`.

Reply:
I missed that, thanks